openbb_terminal/stocks/insider/businessinsider_model.py

```diff
@@ -60,14 +60,13 @@ def get_insider_activity(symbol: str) -> pd.DataFrame:
     )
 
     df_insider["Date"] = pd.to_datetime(df_insider["Date"])
-    df_insider = df_insider.set_index("Date")
-    df_insider = df_insider.sort_index(ascending=True)
 
     l_names = list()
     for s_name in text_soup_market_business_insider.findAll(
         "a", {"onclick": "silentTrackPI()"}
     ):
         l_names.append(s_name.text.strip())
     df_insider["Insider"] = l_names
-
+    df_insider = df_insider.set_index("Date")
+    df_insider = df_insider.sort_index(ascending=False)
```

Comment:
```suggestion
    df_insider = df_insider.sort_index(ascending=True)
```

The `insider_activity` view assumes the DataFrame order is ascending. I get an error with this set to `False`

Reply:
Ah, my bad! Stopped the debugging at the return statement.
Thank You @montezdesousa , the view requires data in reverse chronological order.

Reply:
Nw, thanks for the PR, will approve now